prediction-polls/android/app/src/main/AndroidManifest.xml

```diff
@@ -2,7 +2,10 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools">
 
+    <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
```

Comment:
this is a runtime permission, where do we request it in the code? I could not see anything

Reply:
We don't need to request it. You can give it manually in settings of android. Since it is a debug feature right now no need to request the permission. If you don't specify this permission this permission is locked in new android API's that's why I have gave that permission.

Reply:
everytime we install the app from scratch we will need to go to the settings to give the permission. but okay

Reply:
Nope if you don't remove the app reinstall doesnt change this permission as far as I know.